webapp/TargetedMS/js/QCPlotLegendHelper.js

```diff
@@ -37,6 +37,11 @@ if (!LABKEY.targetedms.QCPlotLegendHelper) {
         addString: function (name, isSequence) {
             var dict = isSequence ? this.peptidePrefixDictionary : this.customIonPrefixDictionary;
 
+            // For peptides, only look at the sequence
+            if (isSequence) {
+                name = name.substring(0, name.indexOf(' '));
```

Comment:
In this case, does name always have ' ' ? if name does not have ' ', then the substring will return ' '.

Reply:
Indeed. Thanks. I thought I had tested this correctly via the JS console in the browser but my second argument to substring() was wrong.